beacon_node/network/src/network_beacon_processor/mod.rs

```diff
@@ -866,6 +864,9 @@ impl<T: BeaconChainTypes> NetworkBeaconProcessor<T> {
                     "Reconstruction not required for block"
                 );
             }
+            Err(BlockError::DuplicateFullyImported(_)) => {
+                debug!("Block has already been imported");
```

Comment:
Maybe we should mention that this log is reconstruction related?

```suggestion
                debug!("Block already imported in parallel with reconstruction");
```

Reply:
Yep good idea!